pages/landing/enterprise-eks/pricing/_form.html

```diff
@@ -0,0 +1,99 @@
+<div class="panel panel-default panel-light pricing-help-notification" style="margin-top: 100px">
+  <div class="panel-body">
+    <h1 style="text-align: center;">
+      Request early access.
```

Comment:
Any reason the contact form is embedded in the pricing page rather than separate? IMO, it makes that page a bit too long and hard to follow.

Reply:
Pulled it out to be similar to all the other pages.